Lesson22/sahra/accordion/script.js

```diff
@@ -0,0 +1,14 @@
+const accordionItemHeaders = document.querySelectorAll(".accordion-item_header");
+accordionItemHeaders.forEach(accordionItemHeader =>{
+  accordionItemHeader.addEventListener("click", event => {
+    accordionItemHeader.classList.toggle("active");
+    });
+});
+/*this didnt work also thats why i did it like in the css line 68
+const accordionItemContents = document.querySelectorAll(".accordion-item_body-content");
```

Comment:
This would not work as by default you only have the header visible and body is hidden.